src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnDecoder.BitString.cs

```diff
@@ -41,15 +41,15 @@ public static partial class AsnDecoder
         ///   <paramref name="ruleSet"/> is not defined.
         /// </exception>
         /// <exception cref="AsnContentException">
-        ///   the next value does not have the correct tag.
+        ///   The next value does not have the correct tag.
```

Comment:
These might need `<para>` tags so they remain on separate lines in the docs, right @carlossanlop?

Reply:
I see now that that would mean a lot of changes throughout this PR. I'm also fine with not adding them.

Reply:
The porttotripleslash tool did not put \<para\> there, so I assumed it was not required.